DrinkManagerConsole/Program.cs

```diff
@@ -1,9 +1,38 @@
-﻿namespace DrinkManagerConsole
+﻿using System;
+
+namespace DrinkManagerConsole
 {
     internal class Program
     {
         private static void Main(string[] args)
         {
+            var exitProgram = false;
+            do
+            {
+                var menu = new Menu();
+                menu.Show();
+                var choice = menu.GetUserChoice();
+
+                switch (choice)
+                {
+                    case MenuChoice.FindByName:
+                        break;
+                    case MenuChoice.FindByAlcoholContent:
+                        break;
+                    case MenuChoice.AddCustomDrink:
+                        break;
+                    case MenuChoice.FindByIngredient:
+                        break;
+                    case MenuChoice.UpdateDrinksFromFile:
+                        break;
+                    case MenuChoice.Exit:
+                        exitProgram = true;
+                        break;
+                    default:
+                        throw new ArgumentOutOfRangeException();
```

Comment:
Is it necessary to put default here? I mean GetUserChoice() won't let any other arguments except those mentioned in cases.

Reply:
Still this default being or not being here doesn't change the fact that program runs like it should without problems.